service/pkg/config/config.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/spf13/viper"
 )
 
+const rootKeyField = "root_key"
```

Comment:
There are additional fields we should ensure are not logged:
* `DB` has a `Password` field that should be redacted
* `SDKConfig` has `ClientSecret`

Another related struct that can get tied to config:
* StandardConfig struct in service/internal/security/standard_cryto.go (contains keys)